src/OSPSuite.UI/Binders/ValueOriginBinder.cs

```diff
@@ -73,7 +74,7 @@ private void onToolTipControllerGetActiveObjectInfo(object sender, ToolTipContro
 
       private void closeUp(CloseUpEventArgs e)
       {
-         var cancel = e.CloseMode == PopupCloseMode.Cancel;
+         var cancel = e.CloseMode != PopupCloseMode.Normal;
```

Comment:
WE now have different mode of closing. Normal means OK is clicked (going to add this to the code as comment in fact)